config/code_schools.yml

```diff
@@ -438,6 +438,13 @@
   va_accepted: false
   full_time: true
   hardware_included: false
-  
-    
-  
+
+- name: Codeup
+  url: http://codeup.com/
+  address1: 600 Navarro St., #350
```

Comment:
In my terminal, it says the server is working but it won't show anything in my browser so I don't know. It seemed a simple enough change so I wasn't worried. None of the other listings are in quotes--but Codeup should be? The keys and values or just the values? I don't understand (really this time).